<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QControlsWidgetClass</class>
 <widget class="QWidget" name="QControlsWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QControlsWidget</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="radiusLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="radiusLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Influence radius</string>
   </property>
  </widget>
  <widget class="QLabel" name="isovalueLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Isovalue</string>
   </property>
  </widget>
  <widget class="QLabel" name="xLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="isovalueLineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="xLineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="objectLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Object</string>
   </property>
  </widget>
  <widget class="QPushButton" name="applyPushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>320</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="yLineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="zLineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="yLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QLabel" name="zLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Z</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="modeSelectGroup">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>151</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Display mode</string>
   </property>
   <widget class="QRadioButton" name="normalModeRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>114</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Normal</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="tensionModeRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>114</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Tension</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
